packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -501,7 +501,23 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
           /* @conditional-compile-remove(reaction) */
           reaction={localParticipant.reaction}
           /* @conditional-compile-remove(spotlight) */
+          spotlightedParticipantUserIds={spotlightedParticipants}
+          /* @conditional-compile-remove(spotlight) */
           isSpotlighted={isSpotlighted}
+          /* @conditional-compile-remove(spotlight) */
+          onStartSpotlight={
+            localParticipant.capabilities?.spotlightParticipant?.isPresent ? onStartSpotlight : undefined
+          }
+          /* @conditional-compile-remove(spotlight) */
+          onStopSpotlight={onStopSpotlight}
```

Comment:
The local participant should always be able to exit spotlight if they are spotlighted regardless of the spotlightParticipant capability